cli.js

```diff
@@ -170,6 +171,56 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
   }, async (argv) => {
     await handleCommand(argv, listHandler);
   })
+  .command('triggers', 'Manages triggers', yargs0 => yargs0
+      .demandCommand()
+      .command('upsert', 'Creates a new trigger or updates an existing one', (yargs1) => {
+        yargs1
+          .usage('Usage: $0 triggers upsert <trigger name> [options]')
+          .positional('trigger name', {
+            describe: 'Trigger name',
+            type: 'string',
+          })
+          .option('source-uri', {
```

Comment:
maybe just "source"?

Reply:
It would be nice to support the verbose trigger format, uri is just one way to get the source in

Reply:
lets try to get @avnerbraverman opinion (as this is a product issue) before we make 10k iteration over flag names...